test/module/irohad/model/CMakeLists.txt

```diff
@@ -57,11 +61,13 @@ target_link_libraries(json_query_factory_test
     model_generators
     json_model_converters
     test_logger
+    sha3_hash
```

Comment:
please add `sha3_hash` dep to `model_generators` lib and remove it from here & targets like this